### DISCUSSION TITLE
Vercel Blob

### DISCUSSION THREAD

**Discussion by IVEN21** [vercel/community#6241] [Feature Question] [answered]
### Question

Vercel's Pro plan, it offers 10GB for blob file storage, which sounds good, but it only gives you 1GB in bandwidth. That's a problem because whenever I stored the image in Blob and access images in the future, it eats upmy bandwidth. I tried it out with just 85MB of files, and already I'm hitting 1.3GB in bandwidth usage. So basically, the plan's storage-to-bandwidth ratio is way off, making it pretty frustrated for me and others. Luckily its still in beta and I don't have to pay the ran out, I hope Vercel team can adjust this.

**Comment by vvo** [comment, marked as answer]
Hey @IVEN21, we just made two changes to offer a better Blob bandwidth package:
- Starting today, Hobby customers have 5GB of free monthly Vercel Blob bandwidth (before: 100 MB)
- Starting today, Pro and Pro trial customers have 10GB of free monthly Vercel Blob bandwidth (before: 1GB)

Hopefully, these new limits will provide better defaults so people can have a good experience while testing Vercel Blob.
Our previous limits (especially for Hobby customers) were too low, sorry about that! 👍.
